gamehub-android/game/src/main/aidl/com/farsitel/bazaar/game/IGameHub.aidl

```diff
@@ -4,13 +4,31 @@ interface IGameHub {
 
     boolean isLogin();
 
-    Bundle getTournamentTimes(String packagename);
+    Bundle getTournamentTimes(String packageName);
 
-    Bundle startTournamentMatch(String packagename, String matchId, String metaData);
+    Bundle startTournamentMatch(String packageName, String matchId, String metaData);
 
     Bundle endTournamentMatch(String sessionId, float score);
 
     Bundle getLeaderboard(int tournamentId);
 
-    Bundle getCurrentLeaderboard(String packagename);
+    Bundle getCurrentLeaderboard(String packageName);
+
+    /**
+     * Notify Bazaar when event id done by user
+     * @params: id of the event
```

Comment:
There is a convention you must mention the name of the parameter and with a dash separate description.
Example:
@params: eventId - the id of a event